app/views/licences/tabs/set-up.njk

```diff
@@ -0,0 +1,52 @@
+{% from "macros/charge-version-status-tag.njk" import statusTag %}
+
+<h2 class="govuk-heading-l">Licence set up</h2>
+
+<h3 class="govuk-heading-m"> Charge information </h3>
+
+{% if chargeInformation.length == 0 %}
+  <p class="govuk-body"> No charge information for this licence.</p>
+{% endif %}
+
+{% if chargeInformation.length > 0 %}
+  <table class="govuk-table">
+    <thead class="govuk-table__head">
+    <tr class="govuk-table__row">
+      <th scope="col" class="govuk-table__header">Start date</th>
+      <th scope="col" class="govuk-table__header">End date</th>
+      <th scope="col" class="govuk-table__header">Reason</th>
+      <th scope="col" class="govuk-table__header">Status</th>
+      <th scope="col" class="govuk-table__header">Action</th>
+    </tr>
+    </thead>
+    <tbody class="govuk-table__body">
+    {% for chargeVersion in chargeInformation %}
+      <tr class="govuk-table__row">
+        <td class="govuk-table__cell">{{ chargeVersion.startDate }}</td>
+        <td class="govuk-table__cell">{{ chargeVersion.endDate }}</td>
+        <td class="govuk-table__cell">{{ chargeVersion.reason }}</td>
+        <td class="govuk-table__cell">{{ statusTag(chargeVersion.status) }}</td>
+        <th class="govuk-table__cell">
+          {% for chargeVersionAction in chargeVersion.action %}
+            {% if not loop.first %} | {% endif %}
+            <a href="{{ chargeVersionAction.link }}" class="govuk-link">
+              {{ chargeVersionAction.text }} </a>
+          {% endfor %}
+        </th>
+      </tr>
+    {% endfor %}
+    </tbody>
+  </table>
```

Comment:
Is there something blocking us from creating the table using the [Design system component](https://design-system.service.gov.uk/components/table/)?

There are times we can't because what we've been asked to reproduce is to complex. But where we can we favour using the components because we believe it gives us an easier upgrade path.